tools/single_use/Cleanup_multiple_firstVisits.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * This script is written for a one time use only to clean up existing data in the
+ * 'session' table where there are multiple first visits, i.e. VisitNo=1,
+ * for a candidate.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Zaliqa Rosli <zaliqa.rosli@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+require_once __DIR__ . '/../generic_includes.php';
+
+$confirm = false;
+if (isset($argv[1]) && $argv[1] === "confirm") {
+    $confirm = true;
+}
+
+$db = \Database::singleton();
+
+// Find CandIDs where there are multiple first visits
+$query_candID = "SELECT CandID 
+                  FROM `session` 
+                  WHERE VisitNo=1 
+                  GROUP BY CandID 
+                  HAVING COUNT(CandID) > 1";
+$candIDs      = $db->pselect($query_candID, array());
+
+if (empty($candIDs)) {
+    echo "No multiple first visits.\n";
+} else {
+    foreach ($candIDs as $entry) {
+        // Order session IDs by date of visit
+        echo "Multiple first visits detected for $entry[CandID]\n";
```

Comment:
```suggestion
        echo "Multiple first visits detected for CandID $entry[CandID]\n";
```

can i be annoying and add this? besides that, its all good